leverage/modules/auth.py

```
@@ -155,3 +155,121 @@ def refresh_layer_credentials(cli):
+    if not sso_profiles:
+        logger.warning("No SSO account profiles found. Run 'leverage aws configure sso' first.")
+        return
+
+    logger.info(f"Found {len(sso_profiles)} account(s) to refresh.")
+
+    client = boto3.client("sso", region_name=cli.sso_region_from_main_profile)
+
+    # Get the SSO access token
+    try:
+        access_token = cli.get_sso_access_token()
+    except Exception as e:
```

Ruff (0.14.6)

Do not catch blind exception: `Exception`

(BLE001)